rush.json

```diff
@@ -1,11 +1,7 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.62.1",
-  "pnpmVersion": "6.30.0",
-  "pnpmOptions": {
-    "strictPeerDependencies": true,
-    "useWorkspaces": true
-  },
```

Comment:
why did we remove `pnpmOptions`?

Reply:
There is a new [pnpm-config.json](https://github.com/aws-solutions/solution-spark-on-aws/blob/rush-update/common/config/rush/pnpm-config.json#L20-L34) files where these options can be configured. It helps to keep rush.json file clean